zephyr/samples/coredumps/boards/frdm_rw612.conf

```diff
@@ -1,7 +1,11 @@
 CONFIG_WIFI_NXP=y
 CONFIG_NXP_RW610=y
 CONFIG_ETH_DRIVER=n
-#fix buffer size for RW612 on Zephyr 4.1.0
-CONFIG_NET_BUF_DATA_SIZE=1744
+
+# Fix buffer size for RW612 on Zephyr 4.1.0+
+CONFIG_NET_BUF_DATA_SIZE=256
```

Comment:
I found out experimentally that it's enough to double the size of a single buffer (there are multiple buffers used).